web/beacon-app/src/features/onboarding/components/StepButtons.tsx

```diff
@@ -2,16 +2,24 @@ import { Trans } from '@lingui/macro';
 import { Button } from '@rotational/beacon-core';
 import React from 'react';
 
+import { userLoader } from '@/features/members/loaders';
 import { useOrgStore } from '@/store';
 
 import { ONBOARDING_STEPS } from '../shared/constants';
+import { isInvitedUser } from '../shared/utils';
 type StepButtonsProps = {
   isSubmitting?: boolean;
   isDisabled?: boolean;
 };
 const StepButtons = ({ isSubmitting, isDisabled }: StepButtonsProps) => {
   const state = useOrgStore((state: any) => state) as any;
   const { currentStep } = state.onboarding as any;
+  const { member } = userLoader();
+  const isInvited = isInvitedUser(member);
+  const shouldDisplayBackButton =
+    currentStep !== ONBOARDING_STEPS.ORGANIZATION ||
```

Comment:
It seems that the back button would continue to appear for invited users because `currentStep !== ONBOARDING_STEPS.ORGANIZATION` is `true`, because they're directed immediately to step 3. Should this be re-arranged so that the following is checked first?
```
(currentStep > 2 && isInvited && currentStep !== ONBOARDING_STEPS.NAME)
```